packages/boxel-ui/addon/components/input/search-input/index.gts

```diff
@@ -44,6 +45,10 @@ const SearchInput: TemplateOnlyComponent<Signature> = <template>
         @state={{if @state @state 'initial'}}
         @errorMessage={{@errorMessage}}
         @placeholder={{if @placeholder @placeholder 'Search'}}
+        style={{cssVar
+          input-height='var(--search-input-height)'
```

Comment:
I changed this to `1000px` and it didn’t change anything but I’m reluctant to remove it without understanding what it’s meant to do, any ideas @lukemelia?

Reply:
Not needed. It's handled in the stylesheet. I'll remove